tyk-single-dc/Chart.yaml

```diff
@@ -34,4 +34,4 @@ dependencies:
   - name: tyk-enterprise-portal
     version: 1.0.0-beta4
     repository: file://../components/tyk-enterprise-portal
-    condition: global.components.enterprisePortal
+    condition: global.components.devPortal
```

Comment:
given that tyk-single-dc has been renamed to tyk-stack would it make sense for the "old umbrella chart" to depend now on the "new portal" ?

Reply:
We have kept `tyk-single-dc` chart just to deprecate them. After that we will delete this chart.